.github/workflows/reusable-python-publish_to_artifactory.yml

```diff
@@ -0,0 +1,72 @@
+name: Publish condan package to Artifactory 
+
+on:
+  workflow_call:
+    inputs:
+        package_name:
+            description: 'Name of the package to build'
+            required: true
+            type: string
+        python_ver:
+            description: 'Python version to use (eg: 3.10)'
+            required: true
+            type: string
+        version_tag:
+            description: 'Version tag of the package to build'
+            required: true
+            type: string
+    secrets:
+        EXTRA_REPO_TOKEN:
+            description: 'Bearer token for the extra repository on JFrog Artifactory'
+            required: true
+
+defaults:
+    run:
+        shell: 'bash -l {0}'
+
+jobs:
+    release_on_artifactory:
+        name: Release package on Artifactory
+        runs-on: ubuntu-latest
+        env:
+            PYTHONUTF8: 1
+            CONDA_CHANNEL_PRIORITY: strict
+            PIP_NO_DEPS: 1 # all dependencies are installed from conda
+        steps:
+            -   name: Checkout
+                uses: actions/checkout@v2
+            -   name: Setup CONDA_LOCK_ENV_FILE
+                run: |
+                    os_conda=$( [[ $RUNNER_OS == "Linux" ]] && echo "linux" || ([[ $RUNNER_OS == "Windows" ]] && echo "win" || echo "osx"))
```

Comment:
**though:** could `reusable-python-pytest_unix_os.yml` and `reusable-python-pytest_windows.yml` be merged together with a similar approach?

how much do they differ beyond using a different OS?

Reply:
if it makes sense, @SophieCurinier, can you plan for a tech issue merging reusable-python-pytest_unix_os.yml and reusable-python-pytest_windows.yml ?
No rush, so you could collect a few improvement issues on the shared workflows, before touching all the repos to use the newer version of these reusable workflows